tests/framework/installer/ceph_csi_templates.go

```diff
@@ -227,6 +227,21 @@ const (
                   readOnly: true
                 - name: ceph-csi-config
                   mountPath: /etc/ceph-csi-config/
+            - name: liveness-prometheus
```

Comment:
can we add a test case to check we are getting expected metrics?

Reply:
https://github.com/rook/rook/blob/master/tests/integration/ceph_csi_test.go#L47

Reply:
we could be as i said before the current testing related to csi in rook in minimum and adding a test like this seems unnecessary given the current test only sees if the pods are running.